ibtk/include/ibtk/muParserRobinBcCoefs.h

```diff
@@ -223,9 +223,9 @@ class muParserRobinBcCoefs : public SAMRAI::solv::RobinBcCoefStrategy<NDIM>
     /*!
      * The mu::Parser objects which evaluate the data-setting functions.
      */
-    std::vector<mu::Parser> d_acoef_parsers;
-    std::vector<mu::Parser> d_bcoef_parsers;
-    std::vector<mu::Parser> d_gcoef_parsers;
+    std::vector<mu::Parser> d_acoef_parsers = std::vector<mu::Parser>(2 * NDIM);
```

Comment:
Is there some way to avoid repeating the typename here?

Reply:
We could instead use the brace initialization
`std::vector<mu::Parser> d_acoef_parsers{ 2*NDIM };`
but I chose to use consistent notation for initialization instead of compactness. Which would you prefer?